inspirehep/modules/records/mappings/records/journals.json

```diff
@@ -181,6 +181,17 @@
                         "journalautocomplete"
                     ],
                     "type": "string"
+                },
+                "short_titles": {
+                    "properties": {
+                        "title": {
+                            "copy_to": [
+                                "journalautocomplete"
+                            ],
+                            "type": "string"
+                        }
+                    },
+                    "type": "object"
```

Comment:
As I was saying yesterday, amending the mappings now is somewhat of a fool's errand, because the schemas are still in flux. On the other hand, the test that you're going to add is going to enforce that this feature is preserved, so it's okay.